server/athenian/api/slogging.py

```diff
@@ -0,0 +1,281 @@
+import argparse
+import codecs
+import datetime
+import functools
+import io
+import json
+import logging
+import os
+import re
+import sys
+import threading
+import traceback
+from typing import Callable, Dict, Sequence, Tuple, Union
+
+import numpy
+import xxhash
+import yaml
+
+
+logs_are_structured = False
+
+
+def get_datetime_now() -> datetime:
+    """Return the current UTC date and time."""
+    return datetime.datetime.now(datetime.timezone.utc)
+
+
+def get_timezone() -> Tuple[datetime.tzinfo, str]:
+    """Discover the current time zone and it's standard string representation."""
+    dt = get_datetime_now().astimezone()
+    tzstr = dt.strftime("%z")
+    tzstr = tzstr[:-2] + ":" + tzstr[-2:]
+    return dt.tzinfo, tzstr
+
+
+timezone, tzstr = get_timezone()
+_now = get_datetime_now()
```

Comment:
`_fest` is broken if we do not restart for a month since it's set only once.

Reply:
The added module docstring should resolve this question.

Reply:
Makes sense, I'll avoid running locally the api for more than a month then :trollface: